README.md

```diff
@@ -32,10 +32,10 @@ npx install-peerdeps @yext/search-ui-react
 ```
 The command will work with Yarn so long as NPM 6+ is installed on the machine.
 
-Once the library and its peer dependencies are installed, the components can be rendered using React by placing them inside the `SearchHeadlessProvider` with the appropriate credentials:
+Once the library and its peer dependencies are installed, the components can be rendered using React by placing them inside the `SearchHeadlessProvider`. `SearchHeadlessProvider` requires a `SearchHeadless` instance, which is created using `provideHeadless(...)` with the appropriate credentials:
```

Comment:
Small nit: I'd say something like "Once the library and its peer dependencies are installed, our React Components should be nested inside the `SearchHeadlessProvider`". I think that's a bit more accurate and concise. React Components implies Components handled by React. The current wording makes it sound a bit like the Components won't render unless there's a `SearchHeadlessProvider`. I'm not sure this is true for all our Components. Some might render, they just would bomb out on user interaction.